src/tasks/harbor/harbor.ts

```diff
@@ -98,29 +105,83 @@ function setAuth(secret): void {
   robotApi.setDefaultAuthentication(bearerAuth)
 }
 
+async function getProjectId(team: string): Promise<string> {
+  const project = (await doApiCall(errors, `Get project for team ${team}`, () =>
+    projectsApi.getProject(team),
+  )) as Project
+  if (!project) return ''
+  const projectId = `${ensure(project.projectId)}`
```

Comment:
Why store it in variable? Could return immediately right?

Reply:
done

Reply:
also, `ensure(project.projectId)` is not needed since existence check happened before

Reply:
No, only project is checked. But not whether or not projectId is set within project